src/coreclr/tools/r2rdump/R2RDump.cs

```diff
@@ -434,7 +434,9 @@ public void Dump(ReadyToRunReader r2r)
                     {
                         perfmapPath = Path.ChangeExtension(r2r.Filename, ".r2rmap");
                     }
-                    PerfMapWriter.Write(perfmapPath, _options.PerfmapFormatVersion, ProduceDebugInfoMethods(r2r), ProduceDebugInfoAssemblies(r2r), r2r.TargetOperatingSystem, r2r.TargetArchitecture);
+                    // TODO: can't seem to find any place that surfaces the ABI. This is for debugging purposes, so may not be as relevant to be correct.
+                    TargetDetails details = new TargetDetails(r2r.TargetArchitecture, r2r.TargetOperatingSystem, TargetAbi.CoreRT);
```

Comment:
This might merit @MichalStrehovsky's attention. I _think_ that CoreRT is basically the NativeAOT ABI which is different from CoreCLR ABI, I recall I had to make fixes around that in the early days of Crossgen2 development.

Reply:
Interesting - that is the default ABI getting emitted currently (as least the NodeFactory's Target says so).